Tests/LibC/TestSnprintf.cpp

```diff
@@ -72,9 +72,10 @@ static bool test_single(Testcase<TArg> const& testcase)
 
     // Checking the results:
     bool return_ok = actual_return == testcase.expected_return;
-    bool canary_1_ok = actual.slice(0, SANDBOX_CANARY_SIZE) == expected.slice(0, SANDBOX_CANARY_SIZE);
-    bool main_ok = actual.slice(SANDBOX_CANARY_SIZE, testcase.dest_n) == expected.slice(SANDBOX_CANARY_SIZE, testcase.dest_n);
-    bool canary_2_ok = actual.slice(SANDBOX_CANARY_SIZE + testcase.dest_n, SANDBOX_CANARY_SIZE) == expected.slice(SANDBOX_CANARY_SIZE + testcase.dest_n, SANDBOX_CANARY_SIZE);
+    // FIXME: Propagate errors.
```

Comment:
I don't think this needs a FIXME. If the test crashes, that's what it is. There's already an unchecked `release_value()` above.